packages/datatrak-web/src/views/AccountSettingsPage/PersonalDetailsSection.tsx

```diff
@@ -3,11 +3,133 @@
  * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useCurrentUser } from '../../api';
 import { AccountSettingsSection } from './AccountSettingsSection';
+import { Button } from '../../components';
+import { TextField } from '@tupaia/ui-components';
+
+const ActionButton = styled(Button)`
+  grid-column: -2;
+`;
+
+const StyledTextField = styled(TextField)`
+  margin: 0; // Use gap on parent to control spacing
+`;
+
+const PersonalDetailsForm = styled.form`
+  display: grid;
+  gap: 1.56rem 1.25rem;
+  max-width: 44.25rem;
+  width: 100%;
+  ${({ theme }) => theme.breakpoints.up('sm')} {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .MuiFormLabel-root {
+    color: ${props => props.theme.palette.text.primary};
+    font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+  }
+`;
 
 export const PersonalDetailsSection = () => {
+  const user = useCurrentUser();
+  const [firstNameValue, setFirstNameValue] = useState(user.firstName ?? '');
+  const [lastNameValue, setLastNameValue] = useState(user.lastName ?? '');
+  const [contactNumberValue, setContactNumberValue] = useState(user.contactNumber ?? '');
+  const [employerValue, setEmployerValue] = useState(user.employer ?? '');
+  const [positionValue, setPositionValue] = useState(user.position ?? '');
+  const [buttonIsDisabled, setButtonDisabled] = useState(true);
+
+  function detailsHaveChanged(): boolean {
+    return (
+      firstNameValue.trim() === (user.firstName ?? '') &&
+      lastNameValue.trim() === (user.lastName ?? '') &&
+      contactNumberValue.trim() === (user.contactNumber ?? '') &&
+      employerValue.trim() === (user.employer ?? '') &&
+      positionValue.trim() === (user.position ?? '')
+    );
+  }
```

Comment:
Is there a preference between this and arrow function notation? I did a ⌘F and it seems like `=>` is more common in our codebase. This reads more clearly to me, but happy to change if we have a convention!